app/src/androidTest/java/com/sdp/movemeet/view/navigation/security/ActivityDescriptionSecurityTest.java

```diff
@@ -1,4 +1,4 @@
-package com.sdp.movemeet.view.navigation.security;
+/*package com.sdp.movemeet.view.navigation.security;
```

Comment:
What's the purpose of the security test classes?

Reply:
The goal is to make sure that when you launch a given class that you are not supposed to be in when you're logged out, and that you are logged out, you're redirected to the Login activity. For some reason, those tests still have a few problems, and since they didn't really have any impact on the overall coverage, I commented them temporarly.

Reply:
I see, could this be refactored as a single method inside the `Navigation` class?

Reply:
Yeah I guess so, we could set the user to null and when navigating, see if the intent is Login. But I'm not sure if that would work correctly as the navigation works only for a logged user. I'll try refactoring that for my next PR :)